src/init.ts

```diff
@@ -155,8 +155,7 @@ async function generateTsConfig(options: Options): Promise<void> {
   const tsconfig = formatJson({
     extends: './node_modules/gts/tsconfig-google.json',
     compilerOptions: {rootDir: '.', outDir: 'build'},
-    include: ['src/*.ts', 'src/**/*.ts', 'test/*.ts', 'test/**/*.ts'],
-    exclude: ['node_modules']
```

Comment:
I had to exclude `node_modules` in one of my projects sometime in the past, but I forgot why.